packages/plugin-dom-editable/src/EventNormalizer.ts

```diff
@@ -1556,7 +1556,7 @@ export class EventNormalizer {
         const x = ev instanceof MouseEvent ? ev.clientX : ev.touches[0].clientX;
         const y = ev instanceof MouseEvent ? ev.clientY : ev.touches[0].clientY;
         let caretPosition = caretPositionFromPoint(x, y);
-        if (!caretPosition || !this._isInEditable(caretPosition.offsetNode)) {
+        if (!caretPosition) {
```

Comment:
*You* are excessive.

Reply:
![image](https://user-images.githubusercontent.com/24205914/87156874-a91bcb80-c2bd-11ea-8488-eb3083562c70.png)